pages/cfs.md

```diff
@@ -1,3 +1,8 @@
 <h1> Continuous Call For Speakers </h1>
 
 <iframe src="https://docs.google.com/forms/d/e/1FAIpQLScnkp-dlCXdg_W5XUV-KdgykCu1lYp87UWLS3FBRddpA0yW_w/viewform?embedded=true" width="760px" height="1600px" frameborder="0" marginheight="0" marginwidth="0" scrolling="no" style="max-width: -webkit-fill-available;">Loading...</iframe>
+
+<html>
```

Comment:
@AugustasV i believe the `<html>...</html>` aren't required here, you can just keep the center and the the button web elements.